trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesGenusOntologyService.php

```diff
@@ -43,20 +56,23 @@ public function __construct(ConfigFactoryInterface $config_factory, ChadoConnect
     $this->config = $config_factory->getEditable($module_settings);
 
     // Chado database.
-    $this->chado = $chado;
+    $this->chado_connection = $chado;
 
     // Tripal Logger service.
     $this->logger = $logger;
   }
 
   /**
-   * Fetch all genus from chado.organism in the host site and construct a genus ontology
-   * configuration values described above. Each genus will contain a configuration value
-   * for trait+unit+method, database and crop ontology.
+   * Fetch all genus from chado.organism in the host site.
+   *
+   * After obtaining orgnism, construct a genus ontology configuration values
+   * where each genus will contain a configuration value for
+   * trait+unit+method, database and crop ontology.
```

Comment:
```suggestion
   * For each organism, initialize an empty template array. 
   * This array will be used by other genus ontology configuration
   * methods, ie. methods in this service.
   *
   * NOTE: There are no actual configuration values being populated here.
```

Reply:
Whenever I have to touch documentation/comments due to coding standards, I personally find that it's a good opportunity to double-check the wording and improve it. This is especially the case when there have been code changes made and the docblock is quite a bit older.

Reply:
I actually really like this note in the doc block as I have been confused on the scope of this method before. For some reason I always think it's doing more then it is and this rewording makes it more clear to me.